opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -217,6 +219,14 @@ message DoubleHistogram {
   AggregationTemporality aggregation_temporality = 2;
 }
 
+message IntSummary {
```

Comment:
Following a previous discussion of the summary metric (in the metrics SIG) and [here](https://github.com/open-telemetry/opentelemetry-proto/pull/199), I believe @bogdandrutu suggested to mark this data format as Legacy. This would help avoid confusion and prevent future instruments from being built on the summary (as we're mainly using it to transport Prometheus summary metrics).

@jmacd @bogdandrutu Would you have any opinions here?

Reply:
I'll accept "Legacy", yes.
Because of this legacy, should we only support Double summaries?
The precise wording can be:

// DoubleSummary metric data are used to convey quantile summaries, 
// a Prometheus and OpenMetrics data type. These data points cannot 
// always be merged in a meaningful way. While they can be useful in some
// applications, histogram data points are recommended for new applications.

Reply:
I've removed `IntSummary` and added the suggested comment the `DoubleSummary`.